libsel4vm/src/arch/arm/vgic/vgic_v2.c

```diff
@@ -292,7 +298,7 @@ int vm_vgic_maintenance_handler(vm_vcpu_t *vcpu)
 }
 
 const struct vgic_dist_device dev_vgic_dist = {
-    .pstart = GIC_DIST_PADDR,
+    .pstart = 0,
```

Comment:
If this structure is a const, but has only dummy data, it feels a bit that it is obsolete now. Can't we remove it?

Reply:
I'd hope so! I agree its kind of pointless now.